### PR TITLE
new guide up

### PR DESCRIPTION
**Pull request by ma-zah**

### CODE REVIEW

File: guides/long-vs-short.mdx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+Why not adopt the same principle in Octomind tests?
+
+## Benefits of chaining short tests together
+
+Octomind is designed with short test cases in mind, and here's why they work so well:
+
+1. **Reusability thanks to chaining:**
+   With Octomind, you can chain test cases together using dependencies. This allows you to reuse prior test cases with just one click. Instead of duplicating steps like login or navigation, you build on existing tests seamlessly.
+2. **Clear overview of test dependencies:**
+   Our dependency view gives you a visual representation of your test suite. Each short test case contributes to a larger coverage tree, making it easy to see which parts of your app are tested—or what still needs attention.
```
Comment (Germandrummer92) [inline guides/long-vs-short.mdx:33]: this sounds like `coverage tree` is a thing (either a feature of us or in general -but both not really true I guess)

File: guides/long-vs-short.mdx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  <img
+    src="/images/expand/generate-more.png"
+    alt="have AI generate more tests"
+  />
+</Frame>
+
+4. **Simple test maintenance:**
+   Short tests are easier to maintain:
+   - Manual updates are quick and contained.
+   - Changes in one test case don't ripple unnecessarily into others.
```
Comment (Germandrummer92) [inline guides/long-vs-short.mdx:55]: I don't really get this point, why would longer test cases lead to this? It's more like change in one _flow_ in your app don't ripple into MULTIPLE test cases, right?

**Review by Germandrummer92 — APPROVED** [review]

File: guides/long-vs-short.mdx (hunk 2 above)
Comment (ma-zah) [inline guides/long-vs-short.mdx:55]:
```suggestion
   - Changes in one flow in your app don't ripple into multiple test cases.
```

File: guides/long-vs-short.mdx (hunk 1 above)
Comment (ma-zah) [inline guides/long-vs-short.mdx:33]:
```suggestion
   Our `dependency view` gives you a visual representation of your test suite. Short test cases and their chaining is clearly displayed, making it easy to see which parts of your app are tested or what still needs your attention.
```

**Pull request merged**